resources/js/Pages/SignUp.vue

```diff
@@ -15,13 +20,12 @@
                     </b-field>
 
                     <b-field label="Email">
-                        <b-input id="email" v-model="form.email" type="email"
-                            maxlength="30">
+                        <b-input id="email" v-model="form.email" type="email">
                         </b-input>
                     </b-field>
 
                     <b-field label="Password">
-                        <b-input id="password" v-model="form.password" type="password" maxlength="30"></b-input>
+                        <b-input id="password" v-model="form.password" type="password" ></b-input>
```

Comment:
```suggestion
                        <b-input id="password" required v-model="form.password" type="password" ></b-input>
```
I think it's better to also add `required` attributes on mandatory inputs, so we check first on the client if the field is empty. I've only done it here, but you'll have to add them to all required field (don't forget the Login view :wink:)